docs/README.md

```diff
@@ -159,11 +162,11 @@ The result of an app analysis.
 | Name | Type | Description |
 | :------ | :------ | :------ |
 | `app` | [`App`](README.md#app) | The app's metadata. |
-| `traffic` | `Record`<`string`, `string`\> | The collected traffic, accessible by the specified name. The traffic is available as a JSON object in the HAR format (https://w3c.github.io/web-performance/specs/HAR/Overview.html). |
+| `traffic` | `Record`<`string`, `Har`\> | The collected traffic, accessible by the specified name. The traffic is available as a JSON object in the HAR format (https://w3c.github.io/web-performance/specs/HAR/Overview.html). |
 
 #### Defined in
 
-[cyanoacrylate/src/index.ts:153](https://github.com/tweaselORG/cyanoacrylate/blob/main/src/index.ts#L153)
+[cyanoacrylate/src/index.ts:187](https://github.com/tweaselORG/hot-glue/blob/main/src/index.ts#L187)
```

Comment:
Something seems off with the `package.json`. This should be cyanoacrylate, shouldn't it?